chandra_aca/plot.py

```diff
@@ -191,45 +201,51 @@ def plot_stars(attitude, catalog=None, stars=None, title=None, starcat_time=None
         stars = agasc.get_agasc_cone(quat.ra, quat.dec,
                                      radius=1.5,
                                      date=starcat_time)
+
     if bad_stars is None:
         bad_stars = bad_acq_stars(stars)
 
     fig = plt.figure(figsize=(5.325, 5.325))
-    ax = fig.add_subplot(1, 1, 1)
-    plt.subplots_adjust(top=0.95)
+
+    # Start with an empty plot that just has the tick labels for yag/zag
+    ax_yz = fig.add_subplot(1, 1, 1)
```

Comment:
It is not actually a lie at the resolution of the screen and eye.  But yes, the transform between row/col and yag/zag is non-linear and you have to plot one or the other in a rectilinear space.  The point is that all the *important* lines are straight in row/col space.